src/render.js

```diff
@@ -76,7 +76,7 @@ module.exports = function render() {
     store.ctx.map.fire('draw.deselect', { featureIds: flushedSelectionSets.deselected });
   }
 
-  store.ctx.map.fire('draw.render');
+  store.ctx.map.fire('draw.render', {});
```

Comment:
This event is undocumented ... is it being used in studio? Should we keep it and document it?